config/forbidden-apis.txt

```diff
@@ -1,5 +1,9 @@
 java.lang.Integer#parseInt(java.lang.String) @ Use oshi.util.ParseUtil#parseIntOrDefault(java.lang.String,int)
+java.lang.Integer#parseInt(java.lang.String,int)
+java.lang.Integer#parseInt(java.lang.CharSequence,int,int,int)
```

Comment:
See [docs](https://jenkins.thetaphi.de/job/Forbidden-APIs/javadoc/signatures-syntax.html) for syntax of multi-line reasons.
```suggestion
@defaultMessage Use existing oshi.util.ParseUtil methods or create a new one

java.lang.Integer#parseInt(java.lang.String)
java.lang.Integer#parseInt(java.lang.String,int)
java.lang.Integer#parseInt(java.lang.CharSequence,int,int,int)
```